preproduction-azure-pipelines.yml

```diff
@@ -35,90 +35,3 @@ jobs:
       verbose: false
   - script: 'npm run spellcheck'
     displayName: 'Run cSpell'
-
-- job: "Syntax"
-  pool:
-    vmImage: 'Ubuntu 16.04'
-  steps:
-  - script: |
-      echo TRAVIS: $(TRAVIS)
-      echo TRAVIS_BRANCH: $(TRAVIS_BRANCH)
-      echo TRAVIS_REPO_SLUG: $(TRAVIS_REPO_SLUG)
-      echo TRAVIS_PULL_REQUEST: $(TRAVIS_PULL_REQUEST)
-      echo TRAVIS_PULL_REQUEST_SLUG: $(TRAVIS_PULL_REQUEST_SLUG)
-      echo TRAVIS_PULL_REQUEST_SHA: $(TRAVIS_PULL_REQUEST_SHA)
-      echo PR_ONLY: $(PR_ONLY)
-    displayName: "Info"
-  - task: Npm@1
-    displayName: 'npm install'
-    inputs:
-      verbose: false
-  - script: 'npm install -D @azure/rest-api-specs-scripts@0.4.0'
-  - script: 'npm test -- test/syntax.js'
-    displayName: 'Syntax validation'
-
-- job: "Semantic"
-  pool:
-    vmImage: 'Ubuntu 16.04'
-  steps:
-  - task: Npm@1
-    displayName: 'npm install'
-    inputs:
-      verbose: false
-  - script: 'npm install -D @azure/rest-api-specs-scripts@0.4.0'
-  - script: 'npm run tsc && node scripts/semanticValidation.js'
-    displayName: 'Semantic Validation'
-
-- job: "ModelValidation"
-  pool:
-    vmImage: 'Ubuntu 16.04'
-  steps:
-  - task: Npm@1
-    displayName: 'npm install'
-    inputs:
-      verbose: false
-  - script: 'npm install -D @azure/rest-api-specs-scripts@0.4.0'
-  - script: 'npm run tsc && node scripts/modelValidation.js'
-    displayName: 'Model Validation'
-
-- job: "BreakingChange"
-  pool:
-    vmImage: 'Ubuntu 16.04'
-  steps:
-  - script: 'scripts/install-dotnet.sh'
-    displayName: 'Install .Net'
-  - task: Npm@1
-    displayName: 'npm install'
-    inputs:
-      verbose: false
-  - script: 'npm install -D @azure/rest-api-specs-scripts@0.4.0'
-  - script: 'npm run tsc && node scripts/breaking-change.js'
-    displayName: 'Breaking Changes'
-
-- job: "LintDiff"
-  variables:
-    NODE_OPTIONS: '--max-old-space-size=8192'
-  pool:
-    vmImage: 'Ubuntu 16.04'
-  steps:
-  - task: Npm@1
-    displayName: 'npm install'
-    inputs:
-      verbose: false
-  - script: 'npm install -D @azure/rest-api-specs-scripts@0.4.0'
-  - script: 'scripts/install-dotnet.sh'
-    displayName: 'install .Net'
-  - script: 'npm run tsc && node scripts/momentOfTruth.js && node scripts/momentOfTruthPostProcessing.js'
-    displayName: 'LintDiff'
-
-- job: "Avocado"
```

Comment:
Did you mean to delete this one?

Reply:
Sorry I didn't notice avocado PR has been merged into master. I will fix it.